include/faabric/util/message.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <faabric/proto/faabric.pb.h>
+
+namespace faabric::util {
+void copyMessage(const faabric::Message* src, faabric::Message* dst);
```

Comment:
I needed a method to deep-copy a `faabric::Message` to set the message in the `faabric::PendingMigrations` from `faabric::BatchExecuteRequest`.

Reply:
Can't you copy messages using assignment i.e. `src = dst`? What's not copied in that case? If it's still needed, I'd put a comment on this method saying why and what's necessary to copy manually. As per comment below, a function like this will rot as soon as someone changes the `Message` and forgets to update it.